apps/ewallet/test/ewallet/web/v1/serializers/websocket_response_serializer_test.exs

```diff
@@ -0,0 +1,96 @@
+defmodule EWallet.Web.V1.WebsocketResponseSerializerTest do
+  use EWallet.Web.SerializerCase, :v1
+  alias EWallet.Web.V1.WebsocketResponseSerializer
+  alias Phoenix.Socket.Reply
+  alias Phoenix.Socket.Message
+
+  describe "serialize/3" do
+    test "serializes a websocket message" do
+      msg = %Message{ref: 1, topic: "topic", event: "event"}
+      res = WebsocketResponseSerializer.serialize(%{something: "cool"}, msg, true)
+
+      assert res == %{
+        event: "event",
+        ref: 1,
+        success: true,
+        topic: "topic",
+        version: "1",
+        data: %{something: "cool"}
+      }
+    end
+  end
+
+  describe "encode!/1 with %Message{}" do
+    test "encodes fields" do
+      msg = %Message{ref: 1, topic: "topic", event: "event", payload: %{}}
+      {:socket_push, :text, encoded} = WebsocketResponseSerializer.encode!(msg)
+      decoded = Poison.decode!(encoded)
+      # refute true
```

Comment:
eh?

Reply:
It's a test to see if you review dee dee :trollface:

Reply:
🙄